search/domain/base.py

```diff
@@ -178,7 +178,6 @@ class Document:
     paper_id_v: str = field(default_factory=str)
     title: str = field(default_factory=str)
     title_tex: str = field(default_factory=str)
-    title_utf8: str = field(default_factory=str)
```

Comment:
Did this get merged into `title` or `title_text`?

Reply:
title